packages/edit-site/src/components/sidebar-dataviews/add-new-view.js

```diff
@@ -99,8 +98,7 @@ function AddNewItemModalContent( { type, setIsAdding } ) {
 					</Button>
 
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
This can be seen when you add a custom data view and click on add new.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="View-before" src="https://github.com/user-attachments/assets/f4b065c0-1206-4c40-b5f2-beae81073b99"> | <img width="419" alt="view-after" src="https://github.com/user-attachments/assets/6779e98c-a202-4eda-9406-88b793a3c97d"> |